vms/platformvm/advance_time_tx_test.go

```diff
@@ -426,43 +403,35 @@ func TestAdvanceTimeTxRemoveSubnetValidator(t *testing.T) {
 		testSubnet1.ID(), // Subnet ID
 		[]*crypto.PrivateKeySECP256K1R{keys[0], keys[1]}, // Keys
 	)
-	if err != nil {
-		t.Fatal(err)
-	}
+	assert.NoError(t, err)
 
 	vm.internalState.AddPendingStaker(tx)
 	vm.internalState.AddTx(tx, status.Committed)
-	if err := vm.internalState.Commit(); err != nil {
-		t.Fatal(err)
-	}
-	if err := vm.internalState.(*internalStateImpl).loadPendingValidators(); err != nil {
-		t.Fatal(err)
-	}
+
+	err = vm.internalState.Commit()
+	assert.NoError(t, err)
+
+	err = vm.internalState.(*internalStateImpl).loadCurrentValidators()
```

Comment:
loadCurrent? or loadPending?

Reply:
You are right. At this point the pending validators should have been loaded.